runtime/src/bank.rs

```diff
@@ -8119,6 +8128,36 @@ impl Bank {
         }
     }
 
+    fn replace_account<O, N>(
```

Comment:
micro-nit: it's best to avoid single characters like `O` for readability, since they can get confused for `0`. maybe use `P` for `previous`?